test/fixtures/eval/eval_typescript.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+require('../../common');
+
+const spawnSync = require('child_process').spawnSync;
+
+const queue = [
+    'enum Foo{};',
+    'throw new SyntaxError("hello")',
+    'const foo;',
+    'let x: number = 100;x;',
+    'const foo: string = 10;',
+    'function foo(){};foo<Number>(1);',
+    'interface Foo{};const foo;',
+    'function foo(){ await Promise.resolve(1)};',
```

Comment:
nit: this seems a bit convoluted

```suggestion
    'async function foo(){ return 1 };',
```

or

```suggestion
    'const foo = async () => 1;',
```

Reply:
Its not the same thing, its convoluted to create a syntax error